_includes/sideblock.html

```diff
@@ -3,6 +3,11 @@
     {% if include.image %}
     <img alt="" src="{{ include.image }}" style="max-width: 100%; margin-bottom: 10px">
     {% endif %}
+    {% if include.external_website %}
+    <div class="mb-3">
+      <a href="{{ include.external_website }}" class="project__resources-itemtitle">{{ t.go_to_website }}</a>
```

Comment:
Personally, I think that `page.external_website` makes sense but I'm not too convinced about this `go_to_website` since it's a CTA that links to an external resource inside a menu that is meant for internal page navigation. Maybe we could insert the link in the logo, WDYT?

Reply:
@libremente yes we could insert the link directly in the logo, no problem. I used an explicit CTA to make it visible, however the link is also specified in the header to keep it visible on mobile.

Reply:
@astagi let's try with the logo link then, I think it could make more sense.
Since it's an external link I'd probably add a `target=_blank` WDYT?
This is slightly related to https://github.com/italia/developers.italia.it/issues/501 though

Reply:
@libremente `target=_blank` FTW  🙌  As mentioned in #501 external links should have `target=_blank`, however related PR contains a lot of conflicts inside the diff, I think we could close it and I'll take care of that task!

cc @bfabio